webpack.common.js

```diff
@@ -46,7 +46,17 @@ module.exports = {
 			commonjs2: 'vue',
 			amd: 'vue',
 			root: 'Vue'
-		}
+		},
+		'@nextcloud/axios': '@nextcloud/axios',
```

Comment:
Can't we automate this somehow?
Instead of manually specifying them?

Reply:
I pushed another commit to exclude all from node_modules. See https://www.npmjs.com/package/webpack-node-externals#why-not-just-use-a-regex-in-the-webpack-config for why the additional package. :wink: 

Seems to still work fine :)

Reply:
Ah yes, I forgot about this https://github.com/skjnldsv/vueexample/blob/74d79f574abea20326369d6a7cedbc7afd7df353/package.json#L78